ManagerThing.md

```diff
@@ -0,0 +1,142 @@
+
+```rb
+{
+"@context": ["http://w3c.github.io/wot/w3c-wot-td-context.jsonld",
+			 { "manager": "http://w3c.github.io/wot/managerthing#" }
+			],
+  "@type": ["ManagerThing"],
+  "name": "ManagerThing",
+  "interaction": [
+    {
+      "@type": ["Action","manager:install"],
+      "name": "install",
+      "inputData":  { "type": "string" },
+      "outputData":  { "type": "string" },
+      "link": [{
+        "href" : "coap://mytemp.example.com:5683/install",
+        "mediaType": "application/json"
+        }]
+    },
+    {
+      "@type": ["Action","manager:run"],
+      "name": "run",
+      "inputData":  { "type": "string" },
+      "link": [{
+        "href" : "coap://mytemp.example.com:5683/run",
+        "mediaType": "application/json"
+        }]
+    },
+    {
+      "@type": ["Action","manager:stop"],
+      "name": "stop",
+      "inputData":  { "type": "string" },
+      "link": [{
+        "href" : "coap://mytemp.example.com:5683/stop",
+        "mediaType": "application/json"
+        }]
+    },
+    {
+      "@type": ["Action","manager:uninstall"],
+      "name": "uninstall",
+      "inputData":  { "type": "string" },
+      "link": [{
+        "href" : "coap://mytemp.example.com:5683/uninstall",
+        "mediaType": "application/json"
+        }]
+    }
+  ]
+}
+```
+
+### 4. WebIDL
+
+The ManagerThing can be described in WebIDL as follows.
+
+```rb
+typedef ScriptID USVString;  // e.g. UUID
+
+interface ScriptManagerThing {
+  // main actions
+  any run(USVString script);  // run a serialized script and return the result or error
+  ScriptID? install(USVString script, optional unsigned long bootSequence);  // save
+  bool uninstall(ScriptID handle);  // uninstall a script by handle (stop, unmark, delete)
+  bool stop(ScriptID handle);  // stop a running script by handle
+};
+```
+
+### 5. Usage
+
+#### 5.1 Preparation for remote management
+Servient supports fixed management command that declared as TD for ManagerThing. The servient exposes API and is controlled by external client through the TD. Client has a program that issues WoT.discover() command to obtain the TD.<br>
+
+<img src=/images/ManagerThing_Fig1.png width=600 alt="Fig.1">
+
+The following diagram depicts how ManagerThing preapard to use.
+
+<img src=/images/ManagerThing1.png width=600 alt="Fig.2">
+
+- ManagerThing registers a TD for ManagerThing to a TD repository.
+- ConsumedThing in an external client acquires the TD by WoT. discover command.
+- ConsumedThing receives the TD and understands the what management capabilities are supported in the ManagerThing.
+
+#### 5.2 Remote management of servient
+
+The external client manages a servient remotely using Scripting API(ConsumedThing).<br>
+- Servient has a ManagerThing and the ManagerThing is accessed from the client remotely.
+- ManagerThing inherited the mechanism from ExposedThing and expose WoT API for ManagerThing.
+
+<img src=/images/ManagerThing_Fig2.png width=600 alt="Fig.3">
+
```

Comment:
IMHO you can simplify this picture: just show a ManagerThing, not the script that creates it (actually it's out of scope for the user how a ManagerThing is created, IMHO it just should be mentioned it's part of the SW image, and it is "there", and discoverable just as the other predefined Things on the device. You could have a separate section _in the end_, illustrating how to do a ManagerThing (by script, or natively part of the runtime).